cmd/promlinter/main.go

```diff
@@ -58,6 +60,7 @@ func main() {
 		Default("false").Short('s').Bool()
 	listPrintAddPos := listCmd.Flag("add-position", "Add metric position column when printing the result.").Default("false").Bool()
 	listPrintAddHelp := listCmd.Flag("add-help", "Add metric help column when printing the result.").Default("false").Bool()
+	listPrintJSON := listCmd.Flag("json", "Print result as json.").Default("false").Bool()
```

Comment:
Instead of a bool for printing out using json format, what do you think about having an `output` flag as a string to be extensible to support different formats like json, markdown, etc